### PR TITLE
integrate star rating & company view

### PR DESCRIPTION
**Pull request by Bacancy-ShubhamSingh**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,21 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap";
+
+
+
+body {
```
Comment (swati-bacancy) [inline app/assets/stylesheets/application.css]: can you please add this css in review.scss

File: app/controllers/companies_controller.rb [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
-  before_action :authenticate_company!, only: %i[index interested_people]
+  before_action :set_company, only: %i[show edit update destroy]
+  before_action :authenticate_company!, only: %i[index interested_people show]
+
+  def index
+    @companies = Company.all
+  end
+
+  def show
+    @company_reviews = CompanyReview.where(company_id: @company.id).order('created_at DESC')
```
Comment (swati-bacancy) [inline app/controllers/companies_controller.rb]: @company.company_reviews

File: app/controllers/companies_controller.rb [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+
+    @avg_review = if @company_reviews.blank?
```
Comment (swati-bacancy) [inline app/controllers/companies_controller.rb]: @company_reviews is giving you blank array?

File: app/views/companies/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+<script>
+  $('.star-rating').raty({
+    path: '/assets',
+    readOnly: true,
+    score: function() {
+      return $(this).attr('data-score');
+    }
+  });
+</script>
```
Comment (swati-bacancy) [inline app/views/companies/show.html.erb]: EOL

File: app/controllers/companies_controller.rb (hunk 3 above)
Comment (Bacancy-ShubhamSingh) [reply app/controllers/companies_controller.rb]: yes in that case when there are no company reviews for the given company

**Pull request merged**